src/streamlink/plugins/tf1.py

```diff
@@ -17,21 +17,23 @@
 log = logging.getLogger(__name__)
 
 
-@pluginmatcher(re.compile(
-    r"https?://(?:www\.)?(?:tf1\.fr/([\w-]+)/direct|(lci)\.fr/direct)/?"
-))
+@pluginmatcher(re.compile(r"""
+    https?://(?:www\.)?(?:tf1\.fr/(?P<live>[\w-]+)/direct/?|
+    (?P<lci>tf1info|lci)\.fr/direct/?|
+    tf1.fr/stream/(?P<stream>[\w-]+))
```

Comment:
The regex should be reformatted. This is barely legible. Literal dots also need to be escaped.